### ISSUE TITLE
`std` doesn't work

### ISSUE THREAD

**Issue by sdwfrost** [nathanaelbosch/ProbNumDiffEq.jl#209]
`std` no longer works on the solution, and throws the following error:

```julia
ERROR: DimensionMismatch: dimensions must match: a has dims (Base.OneTo(2), Base.OneTo(2)), must have singleton at dim 2
```

This is also [in the generated docs](https://nathanaelbosch.github.io/ProbNumDiffEq.jl/stable/getting_started/#Step-3:-Analyzing-the-solution).

**Comment by nathanaelbosch** [comment]
Thanks for raising this issue! It should be fixed once #210 is merged.

In the meantime, since `std(sol.pu[5])` does not work you could manually construct the standard deviations with `sqrt.(diag(cov(sol.pu[5])))` (which is what `std` would do under the hood).

**Comment by nathanaelbosch** [comment]
Just made a new release, so this should now be fixed in the new 0.9.1.
